control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -2524,8 +2524,8 @@ func (r *reconciler) reconcileStorage(ctx context.Context, hcp *hyperv1.HostedCo
 	case hyperv1.AWSPlatform:
 		driverNames = []operatorv1.CSIDriverName{operatorv1.AWSEBSCSIDriver}
 	case hyperv1.OpenStackPlatform:
-		// TODO(stephenfin): Add Manila here once it supports Hypershift
 		driverNames = []operatorv1.CSIDriverName{operatorv1.CinderCSIDriver}
```

Comment:
I would just add a new element to the list.

```go
driverNames = []operatorv1.CSIDriverName{
  operatorv1.CinderCSIDriver,
  operatorv1.ManilaCSIDriver,
}
```

Reply:
done